src/encoded/static/scss/encoded/modules/_visualize.scss

```diff
@@ -46,20 +46,21 @@
     }
 
     @at-root #{&}__assembly {
-        flex: 0 0 40%;
         text-align: right;
         line-height: 30px;
         font-weight: bold;
         font-size: 1.1rem;
+        width: 100px;
+        word-wrap: break-word;
     }
 
     @at-root #{&}__assembly-title {
         @extend .browser-selector__assembly;
         font-size: 1.3rem;
+
```

Comment:
This looks unnecessary too, right?  For now let's air on the side of "best practice" and make as little changes as possible.